components/header/header.module.scss

```diff
@@ -0,0 +1,112 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
+  background: $color-dark;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 5;
+  margin-bottom: 100px;
+}
+
+.header {
+  background: $color-dark;
+  display: flex;
+  height: $header-height;
+  justify-content: space-between;
+  margin: auto;
+  max-width: $max-content;
+  position: sticky;
+  text-align: center;
+
+  @include breakpoint(small) {
+    padding: 0.5rem;
+  }
+}
+
+.logo {
+  display: flex;
+
+  @include breakpoint(medium-only) {
+    margin-left:3.5rem;
+  }
+
+  @include breakpoint(small) {
+    margin-left: 0.5rem;
+  }
+}
+
+.logoText {
+  font-size: 1.25rem;
+  font-weight: 400;
+  margin: 1rem;
+
+  @include breakpoint(small) {
+    margin: 0.5rem;
+  }
+}
+
+.menu {
+  align-items: center;
+  background: $color-dark;
+  display: flex;
+  justify-content: center;
+  opacity: 0.9;
+
+  @include breakpoint(small) {
+    -webkit-transition: all 0.3s ease;
+    flex-direction: column;
+    height: 100%;
+    position: fixed;
+    right: 0;
+    top: -1000px;
+    transition: all 0.3s ease;
+    width: 100%;
+    z-index: -1;
+  }
+
+  &.active {
+    -webkit-transition: all 0.3s ease;
```

Comment:
We don't need browser prefixes as we're only supporting modern browsers and back a couple of versions :)